pros/serial/devices/vex/vex_device.py

```diff
@@ -66,7 +72,7 @@ def _rx_packet(self, timeout: float = 0.01) -> Dict[str, Union[Union[int, bytes,
                 response_header_stack = bytearray(response_header)
                 rx = bytearray()
         if not rx == bytearray(response_header):
```

Comment:
I know this also wasn't touched in this diff, but why did we decide to say `not x == y` instead of `x != y`?

Reply:
I don't know why, and it ain't broke so I'm not going to touch it.